### ISSUE TITLE
Error `PhaseScriptExecution` keeps being thrown without any error message

### ISSUE THREAD

**Issue by gapl** [scribd/Weaver#134]
### Versions

* Weaver version: v1.0.3
* Xcode version: 11.3.1

### Description

We keep getting the following error when building our project:

```
ShellOut encountered an error
Status code: 1
Message: ""
Output: ""
Let the injection begin.
Listing files...
Command PhaseScriptExecution failed with a nonzero exit code
```

### Details

* We have tried with our main project and created new vanilla projects.
* We tried downloading older versions of Weaver. Anything after, including, v1.0.0 doesn't work and produces the same error. Latest working version is v0.12.3.
* Run script phase is included before Compile Sources. It contains:
 
    ```
    weaver swift --config-path .weaver.yaml
    ```

* We have a `.weaver.yaml` file in our project directory. File contents:

    ```
    main_output_path: ProjectName/Generated
    input_paths:
      - ProjectName
    ignored_paths:
      - ProjectName/Generated
      - Carthage
    ```

* The weird thing is, Weaver/Sample project works just fine and compiles fine also with the latest Weaver version. We've compared every inch of the Weaver/Sample project with our own project, as well as with our vanilla test projects and can't seem to find a difference why Weaver/Sample would work and other projects would not.
* Needless to say, we've tried every combinations of settings we could in the `.weaver.yaml` and as command line arguments.

**Comment by gapl** [comment]
Additional investigation revealed, that unless you specify a dependency before building the project with the script phase included, the script phase run will fail.

This should either be fixed or mentioned in the docs.

**Comment by trupin** [comment]
Thanks for filing this @gapl. It does look like a bug. There might already be an open MR for it though.

@happynik, does that look like the bug you solved with #132?

**Comment by happynik** [comment]
Hello.
Yes, that's right, it is this behavior that is fixed in the MR. I just made corrections MR

**Comment by trupin** [comment]
Thanks for confirming @happynik.

Closing this ticket since I just merged #132. A release will follow soon.
